app/dashboards/feature_dashboard.rb

```diff
@@ -61,7 +61,7 @@ class FeatureDashboard < Administrate::BaseDashboard
   # Overwrite this method to customize how features are displayed
   # across all pages of the admin dashboard.
   #
-  # def display_resource(feature)
-  #   "Feature ##{feature.id}"
-  # end
+  def display_resource(feature)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)